shell/platform/embedder/embedder.cc

```diff
@@ -89,16 +89,23 @@ static bool IsRendererValid(const FlutterRendererConfig* config) {
   return false;
 }
 
-#if OS_LINUX || OS_WIN
+#if OS_LINUX
 
 static void* DefaultGLProcResolver(const char* name) {
   static fml::RefPtr<fml::NativeLibrary> proc_library =
       fml::NativeLibrary::CreateForCurrentProcess();
   return static_cast<void*>(
       const_cast<uint8_t*>(proc_library->ResolveSymbol(name)));
 }
+#elif OS_WIN // OS_LINUX
 
-#endif  // OS_LINUX || OS_WIN
+static void* DefaultGLProcResolver(const char* name) {
+  static fml::RefPtr<fml::NativeLibrary> proc_library =
```

Comment:
Is there a reason not to tighten the `#if` ... `#elif` ... `#endif` to be just around the the `fml::NativeLibrary::CreateForCurrentProcess()`/`fml::NativeLibrary::Create("opengl32.dll")` lines?